processing.py

```diff
@@ -21,11 +43,15 @@ def process_run(ref):
     logger = get_run_logger()
     # Grab the BlueSky run
     run = tiled_client_chx[ref]
+    
     # Grab the full uid for logging purposes
     full_uid = run.start["uid"]
     logger.info(f"{full_uid = }")
     logger.info("Do something with this uid")
-    # Do some additional processing or call otehr python processing functions
+    
+    client = from_uri('https://tiled.nsls2.bnl.gov')
+    container = client['chx']['processed']
+    insert_to_tiled(container, run)
```

Comment:
We define the processed tiled client at the top of the file and can use that here to avoid extra network calls.
```suggestion
    insert_to_tiled(tiled_client_processed, run)
```